scene/main/window.cpp

```diff
@@ -1003,6 +1003,7 @@ void Window::_update_window_size() {
 void Window::_update_viewport_size() {
 	//update the viewport part
 
+	Size2i old_size = _get_size();
```

Comment:
Please be aware, that `_get_size` is `Viewport::_get_size()` and returns the size of the Viewport. In certain mode-combinations the viewport size is not affected by the window size, so in these cases the PR in its current form will disable all `NOTIFICATION_WM_SIZE_CHANGED`.

Reply:
You're right. I changed it to storing the old value in a variable and checking against that.